src/config/tolgeerc.ts

```diff
@@ -38,7 +38,8 @@ function parseConfig(rc: any): TolgeeConfig {
   }
 
   if ('projectId' in rc) {
-    if (typeof rc.projectId !== 'number') {
+    const id = Number(rc.projectId);
```

Comment:
This should be assigned to `cfg`, instead of still assigning the raw value from `rc` to ensure the variable holds the correct expected type.